source/input_conv.cpp

```diff
@@ -446,6 +448,7 @@ void Input_Conv::Convert(void)
 //----------------------------------------------------------
 // about exx, Peize Lin add 2018-06-20
 //----------------------------------------------------------
+#ifdef __MPI // liyuanbo 2022/2/23
```

Comment:
Date and author info can be accessed by `git blame`.

Reply:
Thank you for your advices.